.github/ISSUE_TEMPLATE/eol_collection.md

```diff
@@ -14,13 +14,30 @@ When opening PRs, add a link back to this issue for easier tracking.
 -->
 
 - [ ] Write a post to https://community.gazebosim.org/ announcing the community it has last X weeks for finishing their PRs.
+    * <!-- LINK POST HERE -->
```

Comment:
Do we want to keep this empty bullet point under all of the checkboxes? If you look at the rendered preview of the markdown file, it looks a little strange: https://github.com/ignition-tooling/release-tools/blob/9e444fd0fd1aeb8af4f4ba06fc4d71fa96df71ef/.github/ISSUE_TEMPLATE/eol_collection.md

Reply:
I did it on purpose not to forget to add the links there 🙃

Reply:
I'll merge this one now, but feel free to open a PR with formatting suggestions if you're interested, @adlarkin